build.gradle

```diff
@@ -21,7 +21,7 @@ buildscript {
     def roomVersion = System.getenv('CI') ? java7SafeRoomVersion : m1MacSafeRoomVersion
 
     ext.versions = [
-            "kotlin"         : "1.7.0",
+            "kotlin"         : "1.8.0",
```

Comment:
Should we call this out in the CHANGELOG? I know you mentioned there were some issues with the previous version that this bump resolves.

Reply:
Actually yeah, that's a good idea. I'll update the CHANGELOG.